docs/csharp/tutorials/inheritance.md

```diff
@@ -210,21 +210,21 @@ The following example shows the source code for the `Publication` class, as well
 
 - Two publication-related properties
 
-  `Title` is a read-only <xref:System.String> property whose value is supplied by calling the `Publication` constructor, which stores the value in a private field named `pubTitle`.
+  `Title` is a read-only <xref:System.String> property whose value is supplied by calling the `Publication` constructor.
```

Comment:
I made similar changes to this topic yesterday but haven't noticed the second code example with the same issues. In my case, I changed it to say to say the value was stored in a private backing field auto-generated by the compiler. Not sure if we want to add that here too or remove it from where I added.